client_restaurant/shared/src/commonMain/kotlin/presentation/information/RestaurantInfoScreen.kt

```diff
@@ -240,21 +242,24 @@ class RestaurantInfoScreen :
 
     @OptIn(ExperimentalResourceApi::class)
     @Composable
-    private fun LogoutCard(listener: RestaurantInfoInteractionListener) {
+    private fun LogoutCard(
+        onClickLogout: () -> Unit
+    ) {
         BpCard(modifier = Modifier.fillMaxWidth().padding(
             start = Theme.dimens.space16,
             end = Theme.dimens.space16,
             bottom = Theme.dimens.space16
-        ).clickable { listener.onClickLogout() }
-        ) {
+        ).clickable { onClickLogout() }) {
             Row {
                 Icon(
-                    painterResource(Resources.images.logout), modifier = Modifier.padding(
-                        start = Theme.dimens.space16,
-                        top = Theme.dimens.space16,
-                        bottom = Theme.dimens.space16,
-                        end = Theme.dimens.space8
-                    ), contentDescription = null,
+                    painter = painterResource(Resources.images.logout), modifier = Modifier
+                        .padding(
+                            start = Theme.dimens.space16,
+                            top = Theme.dimens.space16,
+                            bottom = Theme.dimens.space16,
+                            end = Theme.dimens.space8
```

Comment:
```suggestion
                            start = 16.dp,
                            top = 16.dp,
                            bottom = 16.dp,
                            end = 8.dp
```

Reply:
Refactoring other's code should has missed edits😵‍💫